mbuild/tests/test_compound.py

```diff
@@ -393,10 +407,10 @@ def test_intermol_conversion2(self, ethane, h2o):
     def test_parmed_conversion(self, ethane, h2o):
         compound = mb.Compound([ethane, h2o])
 
-        structure = compound.to_parmed()
+        structure = compound.to_parmed(box=compound.boundingbox)
```

Comment:
The default right now will be the boundingbox plus a buffer is nothing specified, correct?